test/pylib/scylla_cluster.py

```diff
@@ -964,6 +975,21 @@ async def change_ip(self, server_id: ServerNum) -> IPAddress:
         server.change_ip(ip_addr)
         return ip_addr
 
+    async def change_rpc_address(self, server_id: ServerNum) -> IPAddress:
+        """Lease a new IP address and update conf/scylla.yaml with it. The
+        original IP is released at the end of the test to avoid an
+        immediate recycle within the same cluster. The server must be
+        stopped before its ip is changed."""
+        assert server_id in self.servers, f"Server {server_id} unknown"
+        server = self.servers[server_id]
+        assert not server.is_running, f"Server {server_id} is running: stop it first and then change its ip"
+        self.is_dirty = True
+        ip_addr = IPAddress(await self.host_registry.lease_host())
+        self.leased_ips.add(ip_addr)
```

Comment:
What about the current `server.config["rpc_address"]`?
Shouldn't it be unleased?

Reply:
It will be released automatically after the test.